src/shared/utils/capitalize.js

```diff
@@ -0,0 +1,11 @@
+const capitalize = (string) => {
```

Comment:
Can we use `startCase` from lodash directly? https://lodash.com/docs/4.17.11#startCase

Reply:
Amazing!